### PR TITLE
Add PCAPdroid

### PR DESCRIPTION
**Pull request by emanuele-f**
PCAPdroid is a privacy-friendly app which lets you track, analyze and block the connections made by the other apps in your device. It also allows you to export a PCAP dump of the traffic, inspect HTTP requests/replies, decrypt TLS traffic and much more

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -505,6 +505,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Termux**](https://github.com/termux/termux-app) <sup>**[[F-Droid](https://f-droid.org/app/com.termux)]**</sup>
 * [**Auditor**](https://github.com/GrapheneOS/Auditor)
 * [**Extirpater**](https://gitlab.com/divested-mobile/extirpater) <sup>**[[F-Droid](https://www.f-droid.org/app/us.spotco.extirpater)]**</sup>
+* [**PCAPdroid**](https://github.com/emanuele-f/PCAPdroid) <sup>**[[F-Droid](https://f-droid.org/packages/com.emanuelef.remote_capture)]**</sup>
```
Comment (offa) [inline README.md:508]:
```suggestion
* [**PCAPdroid**](https://github.com/emanuele-f/PCAPdroid) <sup>**[[F-Droid](https://f-droid.org/app/com.emanuelef.remote_capture)]**</sup>
```

**Review by offa — COMMENTED** [review]
Thanks :+1:. Just an editorial note: please use the `app/` url.

**Comment by offa** [comment]
Really nice project, thanks for your work on the project! :smile:

**Comment by emanuele-f** [comment]
Thank you :) You can join the telegram/matrix groups (links in the github page) to get in touch with the community and discuss new features!

**Review by offa — APPROVED** [review]

**Pull request merged**